packages/babel-helpers/src/helpers/jsx.js

```diff
@@ -0,0 +1,54 @@
+/* @minVersion 7.0.0-beta.0 */
+
+let REACT_ELEMENT_TYPE;
```

Comment:
Do we need to worry about using ES6+ here (`let` since the previous code was `var REACT_ELEMENT_TYPE`)? Makes me think of my earlier comment about being able to conditionality compile the helpers based on the config targets?

Reply:
Right, we definitely cannot change this var to let in a refactor. I'll update the ESLint config to make sure that it doesn't happen.